INSTRUCTION.md

```diff
@@ -0,0 +1,13 @@
+## Інструкції з використання
+
+### Запуск додатку за допомогою Docker
+
+1. Побудуйте Docker-образ:
+   ```sh
+   docker build -t yur11lesyk/devops-todolist:1.1 .
```

Comment:
Ensure that the Docker image tag `yur11lesyk/devops-todolist:1.1` is consistent with the task requirements. The task specifies using the tag `todoapp:1.0.0`. If this is intentional, please clarify in the instructions.